packages/arb-rpc-node/cmd/arb-node/arb-node.go

```diff
@@ -181,20 +181,22 @@ func startup() error {
 			badConfig = true
 			fmt.Printf("Unrecognized RPC mode %s", config.Node.Forwarder.RpcMode)
 		}
-	} else if strings.EqualFold(config.Node.Type, "aggregator") {
+	} else if config.Node.Type() == configuration.AggregatorNodeType {
 		if config.Node.Aggregator.InboxAddress == "" {
 			badConfig = true
 			fmt.Println("Aggregator node needs --node.aggregator.inbox-address")
 		}
-	} else if strings.EqualFold(config.Node.Type, "sequencer") {
+	} else if config.Node.Type() == configuration.SequencerNodeType {
 		// Sequencer always waits
 		config.WaitToCatchUp = true
-	} else if strings.EqualFold(config.Node.Type, "validator") {
-		// Validator always waits
-		config.WaitToCatchUp = true
```

Comment:
Just noting that WaitToCatchUp was removed but I'm assuming it was intentional, since I'm not sure it makes sense for validators

Reply:
Correct, doesn't make any sense for validators to wait